backends/dart_minerva/README.md

```diff
@@ -0,0 +1,12 @@
+# dart_minerva
```

Comment:
Can you update it containing how to run it?

Reply:
OK, I'll add it there.